heron/executor/src/python/heron_executor.py

```diff
@@ -668,6 +683,9 @@ def _get_heron_support_processes(self):
         '--port=%s' % self.shell_port,
         '--log_file_prefix=%s/heron-shell.log' % self.log_dir]
 
+    if self.auto_restart_backpressure_container_time_window > 0:
+      retval[self.heron_shell_ids[self.shard]].append('--secret=%s' % self.topology_id)
+
```

Comment:
If clause is not needed since the secret could be enabled by default in all cases.